### PR TITLE
[INLONG-823] Sync the error code with the change of inlong

### LINKED ISSUE #823: [Improve] Delete deployment instruction about postgresql
### Description

The deployment script only supports mysql

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### LINKED ISSUE #832: [Improve][Doc] Improve the error code for TubeMQ manager
### Description

There is a error code change in https://github.com/apache/inlong/issues/8716

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by liaosunny123**
### Prepare a Pull Request
*(Change the title refer to the following example)*

- Title Example: [INLONG-XYZ][Component] Title of the pull request

*(The following *XYZ* should be replaced by the actual [GitHub Issue](https://github.com/apache/inlong/issues) number)*

- Fixes #832 

### Motivation

Sync the error code with the change of inlong

### Modifications

Sync the error code with the change of inlong

### Verifying this change

- [ ] Make sure that the change passes the CI checks.

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

*(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

File: docs/modules/tubemq/error_code.md [hunk 1]
```diff
@@ -5,33 +5,35 @@ title: Error Code
...
-| Client Error | 419| SERVER_RECEIVE_OVERFLOW | Server receives overflow and need to retry. | For long-term overflow, try to expand the storage instance or expand the memory cache size. ||
-| Client Error | 450| CONSUME_GROUP_FORBIDDEN | Consumer group is forbidden. |||
-| Client Error | 452| SERVER_CONSUME_SPEED_LIMIT| Consumption speed is limited. |||
-| Client Error | 455| CONSUME_CONTENT_FORBIDDEN | Consumption is rejected, including that the consumer group is forbidden to filter consume and The filter `streamId` set does not match the allowed `streamId` set, etc. | Confirm the setting of filter of message.  ||
-| Server Error | 500 | INTERNAL_SERVER_ERROR| Internal server error | Refer to ErrMsg for detail to location the error. |
-| Server Error| 503| SERVICE_UNAVILABLE| Temporary ban on reading or writing for business. | Retry it. ||
-| Server Error| 510| INTERNAL_SERVER_ERROR_MSGSET_NULL | Can not read Message Set. | Retry it. ||
+| ---------- |---------| ---------- | ------- | ---- |
+| Operation Success | 0       | Operation Success| Success. ||
```
Comment (gosonzhang) [inline docs/modules/tubemq/error_code.md:15]: 0 or 200 are all successful, errcode 200 needs to reserved and 0 needs to be supplemented.

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/modules/tubemq/error_code.md [hunk 2]
```diff
@@ -5,33 +5,35 @@ title: 错误码定义
...
-| 客户端错误| 452          | SERVER_CONSUME_SPEED_LIMIT        | 消费被限速                                            | 联系管理员处理，解除限速                                     | 
-| 客户端错误| 455          | CONSUME_CONTENT_FORBIDDEN         | 消费内容拒绝，包括消费组禁止过滤消费，过滤的数据流ID集合与允许的ID集合不一致等 | 对齐过滤消费的设置确认没有问题后，再联系管理员处理   |
-| 服务器侧异常 | 500       | INTERNAL_SERVER_ERROR             | 内部服务器错误                                         | 需要结合错误信息，联系管理员确定问题原因后重试 |
-| 服务器侧异常| 503        | SERVICE_UNAVILABLE                | 业务临时禁读或者禁写                                    | 继续重试处理，如果持续的出现该类错误，需要联系管理员处理     |
-| 服务器侧异常| 510        | INTERNAL_SERVER_ERROR_MSGSET_NULL | 读取不到消息集合                                       | 继续重试处理，如果持续的出现该类错误，需要联系管理员处理     |
+| 错误类别     | 错误码 |      错误标记                     |            含义                                    |              备注                          |
+| ---------- |-----| -------------------------------- | ------------------------------------------------- | ----------------------------------------- |
+| 成功操作    | 0   | SUCCESS                           | 操作成功                                            |                                          |
```
Comment (gosonzhang) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/modules/tubemq/error_code.md:15]: 0 or 200 are all successful, errcode 200 needs to reserved and 0 needs to be supplemented.

**Review by dockerzhang — APPROVED** [review]

File: docs/modules/tubemq/error_code.md [hunk 3]
```diff
@@ -5,33 +5,36 @@ title: Error Code
...
-| Client Error | 450| CONSUME_GROUP_FORBIDDEN | Consumer group is forbidden. |||
-| Client Error | 452| SERVER_CONSUME_SPEED_LIMIT| Consumption speed is limited. |||
-| Client Error | 455| CONSUME_CONTENT_FORBIDDEN | Consumption is rejected, including that the consumer group is forbidden to filter consume and The filter `streamId` set does not match the allowed `streamId` set, etc. | Confirm the setting of filter of message.  ||
-| Server Error | 500 | INTERNAL_SERVER_ERROR| Internal server error | Refer to ErrMsg for detail to location the error. |
-| Server Error| 503| SERVICE_UNAVILABLE| Temporary ban on reading or writing for business. | Retry it. ||
-| Server Error| 510| INTERNAL_SERVER_ERROR_MSGSET_NULL | Can not read Message Set. | Retry it. ||
+| ---------- |---------| ---------- | ------- | ---- |
+| Operation Success | 0       | Operation Success| This is the normal error code that is currently used uniformly ||
+| Operation Success | 0       | Operation Success| This is the original normal error code, this part of the error code is gradually modified to 0, but 200 still means normal ||
```
Comment (gosonzhang) [inline docs/modules/tubemq/error_code.md:16]: 200

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/modules/tubemq/error_code.md [hunk 4]
```diff
@@ -5,33 +5,36 @@ title: 错误码定义
...
+| 客户端错误| 405 | ALL_PARTITION_FROZEN              | 所有可用分区都被冻结                                  | 可用分区已被客户端冻结，需要解冻处理或者等待一段时间再重试                    |
+| 客户端错误| 406 | NO_PARTITION_ASSIGNED              | 当前客户端没有被分配分区进行消费                       | 客户端个数超过分区个数，或者服务器还没有进行负载均衡操作，需要等待并重试             |
+| 客户端错误| 407 | ALL_PARTITION_WAITING              | 当前可用分区都达到了最大消费位置                       | 需要等待再重试                                          |
+| 客户端错误| 408 | ALL_PARTITION_INUSE                | 当前可用分区都被业务使用未释放                         | 需要等待业务逻辑调用confirm接口释放分区，需要等待并再重试                 |
+| 客户端错误| 410 | PARTITION_OCCUPIED                | 分区消费冲突，忽略即可                                 | 内部注册的临时状态，业务接口一般不会遇到该报错                          |
+| 客户端错误| 411 | HB_NO_NODE                        | 节点超时，需要降低操作等待一阵后再重试处理                 | 一般出现在客户端在服务器侧心跳超时，这个时候需要降低操作频率，等待一阵待lib注册成功后再重试处理 | 
+| 客户端错误| 412 | DUPLICATE_PARTITION               | 分区消费冲突，忽略即可                                 | 一般是由于节点超时引起，重试即可                                 |
+| 客户端错误| 415 | CERTIFICATE_FAILURE               | 认证失败,包括用户身份认证,以及操作授权不通过               | 一般是用户名密码不一致,或者操作的范围未授权,需要结合错误详情进行排查              |
```
Comment (healchow) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/modules/tubemq/error_code.md:30]:
```suggestion
| 客户端错误| 415 | CERTIFICATE_FAILURE               | 认证失败，包括用户身份认证，以及操作授权不通过               | 一般由于用户名密码不一致，或者操作未授权，需要结合错误详情排查解决              |
```

**Review by healchow — APPROVED** [review]

**Review by gosonzhang — APPROVED** [review]

**Pull request merged**